src/Chapter06/Listing06.28.DefaultConstructorNoLongerAvailable.cs

```diff
@@ -12,7 +12,7 @@ public static void Main()
         {
             Employee employee;
 
-            asdfasdfasdfdsaf
+             //asdfasdfasdfdsaf
```

Comment:
I had commented this line just to trigger the single error about the ctor missing parameter. Should this actually be removed?